pkg/render/logstorage.go

```diff
@@ -2147,12 +2149,12 @@ func (m managedClusterLogStorage) linseedExternalRolesAndBindings() ([]*rbacv1.C
 		},
 	}
 
-	// Bind the secrets permission to the tigera-fluentd namespace. Other controllers may also bind
+	// Bind the secrets permission to the operator namespace. Other controllers may also bind
```

Comment:
We should update the comment a bit more here - this binding now adds permissions for Linseed to create its public cert secret in the tigera-operator namespace, and doesn't have anything to do with access tokens, so we should rephrase it to say that.

Reply:
We can add to the comment on line 2123 above the clusterrole itself to say "controllers that need Linseed access tokens can bind to this using a role binding in their own namespace"